src/pages/mesh/advanced/logging.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Logging
+description: Learn how to use logs for API Mesh for Adobe Developer App Builder.
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+<InlineAlert variant="info" slots="text"/>
+
+Only edge meshes support logging.
+
+# Logging
+
+API Mesh for Adobe Developer App Builder allows you to use logs to monitor and debug issues with your mesh and its sources. After [getting a list](#display-recent-events) of recent logs by rayID, you can [look up a recent log](#display-a-single-log-by-rayid). Alternatively, you can [export logs in bulk](#export-bulk-logs) or for a specific time range.
+
+## Display recent events
+
+The following command displays the 15 most recent events by rayID and the status of those events:
```

Comment:
It's obvious to our team, but should we give a little more detail about what the rayID is?